pkg/mic/mic.go

```diff
@@ -113,7 +113,8 @@ func NewMICClient(cloudconfig string, config *rest.Config, isNamespaced bool, sy
 	}
 	klog.V(1).Infof("CRD client initialized")
 
-	podClient := pod.NewPodClient(informer, eventCh)
```

Comment:
Do you need any change in the MIC. The MIC would continue to run in Linux, and the NMI would run in Linux and windows.

Reply:
Ideally, there is no change in MIC. However, MIC also use the podclient. We modify the current podclient code pod.NewPodClient to add a new pod channel, which will be used in NMI code.  so we need to make the corresponding change in MIC. The change will never change any function and logic for MIC.

Reply:
I agree. We should not change MIC code. Is there any way to avoid this change?

---
In reply to: [374486614](https://github.com/Azure/aad-pod-identity/pull/496#discussion_r374486614) [](ancestors = 374486614)

Reply:
Yeah. I implement a way to avoid the change.